### ISSUE TITLE
Fix identifier for "Recommendation" CIPM references

### ISSUE THREAD

**Issue by manuel489** [relaton/relaton-bipm#19]
References like *(PV, 37, 30)* should fetch with `CIPM Recommendation 01 (1969)`, not `CIPM Resolution 01 (1969)`.
As they are recommendations. Like indicated in https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-58.yml#L9-L12.

**Comment by ronaldtse** [comment]
To paraphrase the bug here:

CIPM outcomes can be a "Recommendation" or a "Resolution".

The link provided here:
https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-58.yml#L9-L12

Is clear a `type: recommendation`.

Therefore, in accordance with the official pattern provided by BIPM here:
https://github.com/metanorma/metanorma-bipm/issues/164

```
[BODY] [Type] {number} (YEAR)
```

It should be retrieved using the syntax:
```
CIPM Recommendation 1 (1969)
```

And also allowed to be retrieved using this syntax:
```
CIPM Recommendation 1-1969
```

**Comment by andrew2net** [comment]
> CIPM outcomes can be a "Recommendation" or a "Resolution".

@ronaldtse some CIPM outcomes have type "[decision](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-101(II).yml#L12)". Does it mean CIPM outcome should be "Recommendation" if it has `type: recommendation` and "Resolution" if it has any other type?

**Comment by ronaldtse** [comment]
> Metadata doesn't have type attribute

Are we mixing things here?

There are 4 types of BIPM data (see https://github.com/metanorma/bipm-data-importer):
* CGPM outcomes
* CIPM outcomes
* CIPM decisions
* CCTF recommendations

> Does it mean CIPM outcome should be "Recommendation" if it has type: recommendation and "Resolution" if it has any other type?

There are only four outcome types:
- recommendation (https://github.com/metanorma/bipm-data-importer/issues/28)
- resolution (https://github.com/metanorma/bipm-data-importer/issues/13)
- decision (https://github.com/metanorma/bipm-data-importer/issues/28)
- declaration (https://github.com/metanorma/bipm-data-outcomes/issues/7) (this is new)

These outcome types are all specified in the data. There should not be an instance without type.

> So the question is: what type should be documents created from metadata?

I'd like to clarify the terminology here. Relaton does not create "documents" as data, it creates bibliographic items that allow reference.

These are the things that can be referred to in Relaton-BIPM:

* meetings (CIPM meetings, CGPM meetings)
* outcomes (recommendation, resolution, decision, declaration)
* documents (SI Brochure and other documents in bipm-si-brochure)

In order for these things to be referenced, we need to create them in relaton-data-bipm.

This also means that every outcome should have its own file in relaton-data-bipm.

**Comment by andrew2net** [comment]
> > Metadata doesn't have type attribute
> 
> Are we mixing things here?
> 
> There are 4 types of BIPM data (see https://github.com/metanorma/bipm-data-importer):
> 
> * CGPM outcomes
> * CIPM outcomes
> * CIPM decisions
> * CCTF recommendations

Saying metadata I meant section `metadata` in the bipm-data-outcomes. For example we create the [document](https://github.com/relaton/relaton-data-bipm/blob/master/data/cipm/decision/102-1.yaml) from the [metadata section](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-102(I).yml#L2-L7). Which type the document should be? Meeting?

> There are only four outcome types:
> 
> * recommendation ([Assign different types to "resolution"s, the `decision` and `recommendation` metanorma/bipm-data-importer#28](https://github.com/metanorma/bipm-data-importer/issues/28))
> * resolution ([Update fetching location of "CGPM Resolutions" metanorma/bipm-data-importer#13](https://github.com/metanorma/bipm-data-importer/issues/13))
> * decision ([Assign different types to "resolution"s, the `decision` and `recommendation` metanorma/bipm-data-importer#28](https://github.com/metanorma/bipm-data-importer/issues/28))
> * declaration ([New type value `type: declaration` for 90th meeting of the CIPM metanorma/bipm-data-outcomes#7](https://github.com/metanorma/bipm-data-outcomes/issues/7)) (this is new)
> 
> These outcome types are all specified in the data. There should not be an instance without type. 

I see the new attribute [`type`](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-102(I).yml#L12) in the [`resolutions`](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-102(I).yml#L8) collection. I was confused by your phrase:

> CIPM outcomes can be a "Recommendation" or a "Resolution".

In the bipm-data-outcomes I found only `recommendation` and `decision` for CIPM body.
So we just use specified type, correct?

**Comment by ronaldtse** [comment]
> I see the new attribute [type](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-102(I).yml#L12) in the [resolutions](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-102(I).yml#L8) collection. 

Yes, as mentioned in those tickets above, the `type` is somewhat new, but not that new (April 13).

> In the bipm-data-outcomes I found only recommendation and decision for CIPM body.
> So we just use specified type, correct

Yes. Just use specified type.

**Comment by ronaldtse** [comment]
> Saying metadata I meant section `metadata` in the bipm-data-outcomes. For example we create the [document](https://github.com/relaton/relaton-data-bipm/blob/master/data/cipm/decision/102-1.yaml) from the [metadata section](https://github.com/metanorma/bipm-data-outcomes/blob/576929b1ab359bfe4397b40865e1cb65cf575a76/cipm/meetings-en/meeting-102(I).yml#L2-L7). Which type the document should be? Meeting?
> 

As mentioned earlier, we need these types:

* meetings (CIPM meetings, CGPM meetings)
* outcomes (recommendation, resolution, decision, declaration)
* documents (SI Brochure and other documents in bipm-si-brochure)

## CIPM outcomes

https://github.com/metanorma/bipm-data-outcomes/tree/main/cipm provides these objects:

* meetings
  * resolutions (decision, recommendation, etc.)

## CGPM outcomes

https://github.com/metanorma/bipm-data-outcomes/tree/main/cgpm

* meetings
  * resolutions (resolution, recommendation, etc.)

## CCTF outcomes

https://github.com/metanorma/bipm-data-outcomes/tree/main/cctf

* meetings
  * resolutions (recommendation, etc.) (there's only one here)

**Comment by ronaldtse** [comment]
Confirm fixed. Thanks @andrew2net !
